### DISCUSSION TITLE
Не делать копию оригинала плейлиста

### DISCUSSION THREAD

**Discussion by EldarAliev** [Chimildic/YaMuTools#45] [Идея] [answered]
Предлагается добавить настройку "Не делать копию оригинала плейлиста", когда выполняется функция "поиск дублей" и "изменение оригинального плейлиста". Если такая функция есть, прошу извинить, я не нашёл.

**Comment by Chimildic** [comment, marked as answer]
Вместо создания плейлиста-копии, названия копируются в буфер обмена с версии 0.8.0. В случае ошибки, плейлист можно восстановить в [импорте](https://music.yandex.ru/import).
